docs/_layouts/default.html

```diff
@@ -0,0 +1,92 @@
+<html>
+<head>
+    <title>{% if page.title %}{{ page.title }} | {% endif %}{{ site.title }}</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <meta name="description" content="{{ site.description }}">
+    <link rel="stylesheet" href="/css/main.css">
+</head>
+<body>
+<section>
+    <section class="content">{{ content }}</section>
+    <section class="nav">
+        <nav>
+            <ul>
+                <li><a href="/" id="logo">☴ Mill</a></li>
+                <li><a href="/getting-started">Getting Started</a></li>
+                <li><a href="/configuration">Configuration</a></li>
+                <li>
+                    <a href="/generate">Generate</a>
+                    <ul>
+                        <li><a href="/generate/documentation">&middot; Documentation</a></li>
```

Comment:
Why doesn't the style for `<li>` give you the `&middot;`. Why the specific request?